externs/olx.js

```diff
@@ -3547,7 +3598,7 @@ olx.layer.VectorOptions.prototype.renderBuffer;
 
 /**
  * Source.
- * @type {ol.source.Vector}
+ * @type {ol.source.Vector|ol.source.VectorTile}
```

Comment:
I personally don't think there is one, but there are the existing TileVector examples. I don't know if anyone really uses that. I'd be more than happy to remove it, along with the existing TileVector examples (these could be changed to use ol.layer.Vector and a tile loading strategy). Removal would also make ol.source.VectorTile simpler, because the compatibility methods for ol.source.TileVector could be removed. See 1409cb20dbae9e182f1c8eaf77f9515f9c59dfb5.